indico/modules/events/layout/controllers/layout.py

```diff
@@ -74,12 +75,24 @@ def _process(self):
 
 
 class RHLayoutEdit(RHLayoutBase):
-    def _process(self):
+    def _process_GET(self):
         if self.event.type_ == EventType.conference:
             return self._process_conference()
         else:
             return self._process_lecture_meeting()
 
+    def _process_POST(self):
+        old_vals = self.event.get_settings('layout')
+        self._process_GET()
+        changes = {name: [old_vals.get(name), value, 'string']
+                   for name, value in self.event.get_settings('layout').items()
+                   if old_vals.get(name) != value}
+        changes['_diff'] = True
+        if changes:
+            log_text = "Layout was updated"
```

Comment:
why did you assign it here to `log_text`?

Reply:
This has been ported almost as-is from a plugin. I didn't change it because I think it helps understanding which text will be ultimately logged, rather than passing the string literal as a positional argument to `self.event.log(...)`.

Reply:
Ok, then at least use consistent type of quotes and replace `"` with `'` :P

Reply:
As far as I remember, we were using double-quote strings for natural language strings so that apostrophes, common in English, don't need to be escaped. I used single-quote strings in the previous natural language strings since they contain `"` inside and I preferred instead of escaping them.

But it's been a while, so let me know what I should stick to.

Reply:
mostly it's just "single quotes unless the string contains single quotes" nowadays ;) it's not *that* likely that a string suddenly gets changed to include single quotes anyway when it didn't before

Reply:
`'fixed'` xP